### PR TITLE
HOTT-880: Remove unused endpoints

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-880

### What?

I have added/removed/altered:

- [x] Removed unused endpoints

### Why?

I am doing this because:

- These were discovered as part of an investigation into the removal of unused authenticated endpoints in V1

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -297,14 +297,14 @@ jobs:
 
   smoketest_dev:
     docker:
-      - image: 'cypress/base:latest'
+      - image: 'cypress/base:16.5.0'
```
Comment (willfish) [inline .circleci/config.yml:300]: The latest tag for cypress/base hasn't been updated in a year (and therefore clearly isn't the latest)

Comment (willfish) [reply .circleci/config.yml:300]: I've updated this since it was out of step with the cypress test dependencies

**Review by jebw — APPROVED** [review]
Approving on the basis the NewRelic logs demonstrate these are unused?

**Comment by alexdesi** [comment]
amazing cleaning!
just double check we do not forget factories, 
- tests (actually I don't think this, they'd fail)
- jsons used for tests
 - (rake) tasks only relevant to V1
 - routes (I think these are all right)

**Comment by willfish** [comment]
@jebw New relic is actually how I enumerated these. I went through every v1 controller and searched for requests for each one in over a month

**Comment by willfish** [comment]
We wouldn't use fixtures, here @alexdesi since we're in a position to be able to rely on factories that integrate with the db. Those factories are used in other places and are controller independent. I've looked for rake tasks and can't find any that run against these endpoints.

**Pull request merged**